util/diff_runs_test.py

```diff
@@ -78,6 +80,28 @@ def results(sha, platform):
         self.assertIn('1 differences', logged)
         self.assertIn('1 tests', logged)
 
+    def test_cull_ignored_tests_dir(self):
```

Comment:
Can we add tests for removes-all and removes-none?

Reply:
Removes none is done as 'test_no_difference', so just test_removes_all added.